cmd/kube-apiserver/app/server.go

```diff
@@ -133,7 +133,10 @@ func Run(s *options.APIServer) error {
 	}
 
 	// Proxying to pods and services is IP-based... don't expect to be able to verify the hostname
-	proxyTLSClientConfig := &tls.Config{InsecureSkipVerify: true}
+	proxyTLSClientConfig, err := transport.TLSConfigFor(s.KubeletConfig.TransportConfig())
```

Comment:
as I remember api-server proxy didn't send certs to kubelet without this fix. And kubelet complained about this. Will recheck this behavior on Monday.